sled-agent/src/config.rs

```diff
@@ -51,6 +51,9 @@ pub struct Config {
     pub sidecar_revision: SidecarRevision,
     /// Optional percentage of DRAM to reserve for guest memory
     pub vmm_reservoir_percentage: Option<u8>,
+    /// Optional DRAM to reserve for guest memory in MiB (cannot be used with
```

Comment:
nit: maybe "mutually exclusive option with `vmm_reservoir_percentage`"

Reply:
Fixed in [23a25d6](https://github.com/oxidecomputer/omicron/pull/4137/commits/23a25d6071e32d2cf4b14cb17cd744d3f30aa814)